initfiles/etc/DIR_NAME/environment.xml.in

```diff
@@ -430,6 +430,25 @@
                   required="Read"
                   resource="WsEclAccess"/>
    </EspBinding>
+   <EspControlBinding defaultForPort="true"
+               defaultServiceVersion=""
+               name="espcontrol"
+               protocol="http"
+               resourcesBasedn="ou=WsEspControl,ou=EspServices,ou=ecl"
```

Comment:
For now, I am not sure that is needed. If you want, you may create a JIRA for ConfigMgr.